osu.Game/osu.Game.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="2.1.1" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
     <PackageReference Include="ppy.osu.Framework" Version="2018.626.0" />
-    <PackageReference Include="SharpCompress" Version="0.21.1" />
+    <PackageReference Include="SharpCompress" Version="0.17.1" />
```

Comment:
You sure this shouldn't be 0.18.1? This is what it was before https://github.com/peppy/osu/commit/563cb46b4aa38773348e8aed6f8367e578352eb3#diff-03a173764c2f60ca41448922be87d964L22

Reply:
I just changed it to the dependency listed in the squirrel nuget package. Either should be fine.